src/verilator/SConscript

```diff
@@ -0,0 +1,7 @@
+Import('*')
```

Comment:
FYI, we need licenses on all of these files. See other files for examples. The copyright is to University Regents of California or something like that. The DArchR/docs repo has the exact wording.